07_errors/README.md

```diff
@@ -0,0 +1,13 @@
+# Lab 7 - Errors
+
+Create an executable go program in directory `07_errors/USERNAME`
+Copy the CRUD puppy from upstream master `06_puppy/USERNAME`
```

Comment:
ditto

Reply:
This one is actually ok, directory to create is `07_errors/USERNAME`

Reply:
My comment was on the `06_puppy` line, which did not match.